packages/block-editor/src/components/inserter/block-types-tab.js

```diff
@@ -30,6 +30,17 @@ const MAX_SUGGESTED_ITEMS = 6;
  */
 const EMPTY_ARRAY = [];
 
+// Blocks that should be prioritized in the Inserter tab.
+const PRIORITIZED_BLOCKS = {
+	theme: [
+		'core/navigation',
+		'core/template-part/header',
+		'core/template-part/footer',
+		'core/query/posts-list',
+	],
+	// design: [],
```

Comment:
We need to decide on which blocks are going to be prioritzed and in what order. --cc @mtias